parcel_test.go

```diff
@@ -31,57 +31,120 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "./tracker.db") // настройте подключение к БД
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	id, err := store.Add(parcel)
+	require.NoError(t, err)
+	require.Greater(t, id, 0)
+
+	parcel.Number = id
 
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	gotParcel, err := store.Get(parcel.Number) // получаем по возвращенному идентификатору
+	require.NoError(t, err)
+	require.Equal(t, parcel.Number, gotParcel.Number)
+	require.Equal(t, parcel.Client, gotParcel.Client)
+	require.Equal(t, parcel.Status, gotParcel.Status)
+	require.Equal(t, parcel.Address, gotParcel.Address)
+	require.Equal(t, parcel.CreatedAt, gotParcel.CreatedAt)
 
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что посылку больше нельзя получить из БД
+	err = store.Delete(parcel.Number)
+	require.NoError(t, err)
+
+	_, err = store.Get(parcel.Number)
+	require.Error(t, err)
+	require.Error(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```